README.md

```diff
@@ -6,63 +6,158 @@
+distribution, the histogram of your $\chi_{PQM}^2$ values should follow the  $\chi^2$
+distribution. The degrees of freedom (DoF) will equal `DoF = num_refs - 1` The
+peak of this distribution will be at `DoF - 2`, the mean will equal `DoF`, and
+the standard deviation will be `sqrt(2 * DoF)`. If your $\chi_{PQM}^2$ values are too
+high (`chi^2 / DoF > 1`), it suggests that the samples are out of distribution.
+Conversely, if the values are too low (`chi^2 / DoF < 1`), it indicates
+potential duplication of samples between `x` and `y` (i.e.
+memorization for generative models).
+
+If your two samples are drawn from the same distribution, then the $\text{p-value}(\chi_{PQM}^2)$
+should be drawn from the random $\mathcal{U}(0,1)$ distribution. This means that if
+you get a very small value (i.e., 1e-6), then you have failed the null
+hypothesis test, and the two samples are not drawn from the same distribution.
+If you get values approximately equal to 1 every time then that suggests
+potential duplication of samples between `x` and `y`.
+
+PQMass can work for any two datasets as it measures the distribution shift between the $x$ and $y$, which we show below.
+
+## Example
+
+We are using 100 regions. Thus, the DoF is 99, our expected $\chi^2$ peak of the distribution is 97, the mean is 99, and the standard deviation should be 14.07. With this in mind, we set up our example. For the p-value, we expect to be between 0 and 1 and a significantly small p-value (e.g., $< 0.05$ or $< 0.01$) would mean we reject the null hypothesis and thus $x$ and $y$ do not come from the same distribution.
+
+Our expected p-value should be around 0.5 to pass the null hypothesis test; any significant deviation away from this would indicate failure of the null hypothesis test.
+
+Given two distributions, $x$ and $y$, sampling from a $\mathcal{N}(0, 1)$ in 10 dimensions, the goal is to determine if they come from the same underlying distribution. This is considered the null test as we know they come from the same distribution, but we show how one would use PQMass to determine this.
 
 ```python
 from pqm import pqm_pvalue, pqm_chi2
 import numpy as np
 
-x_sample = np.random.normal(size = (500, 10))
-y_sample = np.random.normal(size = (400, 10))
+p = np.random.normal(size = (500, 10))
+q = np.random.normal(size = (400, 10))
+
+# To get chi^2 from PQMass
+chi2_stat = pqm_chi2(p, q, re_tessellation = 1000)
+print(np.mean(chi2_stat), np.std(chi2_stat)) # 98.51, 11.334
 
 # To get pvalues from PQMass
-pvalues = pqm_pvalue(x_sample, y_sample, num_refs = 100, re_tessellation = 50)
-print(np.mean(pvalues), np.std(pvalues))
+pvalues = pqm_pvalue(p, q, re_tessellation = 1000)
+print(np.mean(pvalues), np.std(pvalues)) # 0.50, 0.26
+```
+
+We see that both $\chi_{PQM}^2$ and $\text{p-value}(\chi_{PQM}^2)$ follow the expected $\chi^2$ indicatiing that both $x$ and $y$ come from the same underlying distribution.
```

Comment:
```suggestion
We see that both $\chi_{PQM}^2$ and $\text{p-value}_{PQM}$ follow the expected $\chi^2$ indicating that both $x$ and $y$ come from the same underlying distribution.
```

Reply:
I am leaving $\text{p-value}(\chi_{PQM}^2)$ as this is how it is noted in Equation 12 in the paper